### ISSUE TITLE
"Fire-and-forget" tasks

### ISSUE THREAD

**Issue by oliver-kriska** [zed-industries/zed#9804]
### Check for existing issues

- [X] Completed

### Describe the feature

It would be nice to have some option to just fire one task and don't care about and OPTION + T will execute previous or main task, not the last one which was "fire-and-forget". 

So basically some example situation. I run test multiple times with task, so I have to just hit OPTION + T, but now I need run migration or formatter/whatever, but I need it only once. So OPTION + SHIFT + T, select mix ecto.migrate and done. Now when I want to run test I have to go back to OPTION + SHIFT + T and select test task. 

Instead with one some config for `ecto.migrate` task  like "fire_and_forget: true" it would be just OPTION + T and it will execute test taks. 

That would be very nice UX/DX for tasks. 

based on discussion here: https://gist.github.com/josevalim/2e4f60a14ccd52728e3256571259d493
